content/projects/gsoc/2020/project-ideas/jenkins-x-openwrt-ppa.adoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: gsocprojectidea
+title: "PPA for OpenWRT"
+goal: "Create a platform to build and host packages for OpenWRT (PPA, or Personal Package Archive) using Jenkins X"
+category: Jenkins X
+year: 2020
+status: published
+skills:
+- Shell
+- Makefiles
+- Docker
+- Kubernetes
+mentors:
+- "zoobab"
+links:
+  web: https://www.openwrt.org
+---
+
+=== Background
+
+OpenWrt is a Linux distribution for off the shelf WiFi routers. People who want
+to make and distribute their personal packages, often for development purposes.
+So far the best option is to build the whole firmware from zero, but very few
+people use the OpenWrt SDKs. The idea behind this project is to build a
+platform to build and host those personal package repositories, possibility
+integrated with GIT farms, so that the end users can add a repositiory URL to
+use those packages, modelled on what Ubuntu PPA (Personal Package Archive) has
```

Comment:
`modelled` is misspelled. Should be `modeled`

Reply:
```suggestion
use those packages, modeled on what Ubuntu PPA (Personal Package Archive) has
```